includes/Admin/Admin_Tools.php

```
@@ -379,6 +379,313 @@ public function tool_page() {
+                        printf(
+                            /* translators: %s: Pro badge */
+                            esc_html__( 'Shortcodes marked with %s require WP User Frontend Pro.', 'wp-user-frontend' ),
+                            '<span style="background: #ff6b35; color: #fff; padding: 2px 6px; border-radius: 3px; font-size: 10px; font-weight: 600; text-transform: uppercase;">Pro</span>'
+                        );
+                        ?>
+                        <a href="https://wedevs.com/wp-user-frontend-pro/" target="_blank" style="margin-left: 5px;"><?php esc_html_e( 'Upgrade to Pro', 'wp-user-frontend' ); ?></a>
+                    </p>
+                </div>
+            <?php endif; ?>
+
+            <?php foreach ( $shortcodes as $category_key => $category ) : ?>
```

PHPMD (2.15.0)

Avoid unused local variables such as '$category_key'. (undefined)

(UnusedLocalVariable)